### ISSUE TITLE
conda release

### ISSUE THREAD

**Issue by znicholls** [ramonhagenaars/nptyping#1]
Hi there,
Thanks for your efforts with nptyping, it's perfect for its use! We want to use it in our project (https://github.com/openclimatedata/openscm) but are also thinking about releasing our work with conda. To do that, all the dependencies need to be available on conda too. Would you be open to releasing nptyping on conda? I'd be happy to do the setup work for you/talk you through it, it's super simple for Python projects like this (once you've seen it done before). 
Cheers,
Zeb

**Comment by ramonhagenaars** [comment]
Hi znicholls! 
Great to hear that ``nptyping`` is of use to you! I know of Conda, but have no experience with publishing to it. I'd be happy to deploy to Conda though. Are there any changes to be made to this repository that you could provide in a pull request?

**Comment by Japanuspus** [comment]
TLDR: Anyone can request the `nptying` is added to the fully hosted `conda-forge` channel by following instructions at [conda-forge.org/#add_recipe](https://conda-forge.org/#add_recipe).

## Background

If you are not a conda user yourself, a bit of context is probably in place. [`conda`](https://docs.conda.io/en/latest/) is the package manager used by the [`anaconda`](https://www.anaconda.com/distribution/) python distribution maintained by continuum.org.
The distribution is freely available for all uses as continuum earns their living from an enterprise solution (no affiliation). Coming to MS Windows from *nix-platforms, conda has been a godsend for handling binary package versions.

## Providing conda package

A conda package is a binary build based on a package specification (*recipe*) with metadata such as source location, dependencies in terms of other conda packages, etc. The recipe is usually maintained outside the package codebase.

Continuum maintains the main package repository (*channel*) for conda. For the included packages, they have created conda package specifications and run builds as source packages are updated.

For packages not in the main channel, there is a community effort at conda-forge.org to provide additional conda packages. conda-forge has infrastructure to build packages.

To request that a package be included in conda-forge, you (or anyone else) should follow the instructions for [adding a recipe](https://conda-forge.org/#add_recipe).

Main advantage of you as package maintainer also creating the conda-forge recipe is that you can easily coordinate dependencies, but if you are not interested in doing so just indicate that in the comments here, and I am sure someone will pick it up.

**Comment by znicholls** [comment]
Hey @Japanuspus thanks for jumping in! @ramonhagenaars sorry I opened this and then no longer had the time to pull it together. If the time reappears I'd still like to but it's not going to be soon.

**Comment by jaimergp** [comment]
If we end up using this in our team I would happily contribute the `conda-forge` package.

**Comment by ramonhagenaars** [comment]
@jaimergp that would be awesome! Im quite busy at the moment, so I'm happy with the help!

**Comment by Jasha10** [comment]
If I'm not mistaken, this issue can be closed now.
`nptyping` is now available on the conda forge.
https://github.com/conda-forge/nptyping-feedstock
